src/main/java/se/sundsvall/emailreader/integration/EWSIntegration.java

```diff
@@ -0,0 +1,109 @@
+package se.sundsvall.emailreader.integration;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import se.sundsvall.emailreader.api.model.Email;
+
+import microsoft.exchange.webservices.data.core.ExchangeService;
+import microsoft.exchange.webservices.data.core.PropertySet;
+import microsoft.exchange.webservices.data.core.enumeration.misc.ExchangeVersion;
+import microsoft.exchange.webservices.data.core.enumeration.property.BasePropertySet;
+import microsoft.exchange.webservices.data.core.enumeration.property.WellKnownFolderName;
+import microsoft.exchange.webservices.data.core.service.folder.Folder;
+import microsoft.exchange.webservices.data.core.service.item.EmailMessage;
+import microsoft.exchange.webservices.data.core.service.item.Item;
+import microsoft.exchange.webservices.data.core.service.schema.FolderSchema;
+import microsoft.exchange.webservices.data.credential.WebCredentials;
+import microsoft.exchange.webservices.data.search.FindItemsResults;
+import microsoft.exchange.webservices.data.search.FolderView;
+import microsoft.exchange.webservices.data.search.ItemView;
+import microsoft.exchange.webservices.data.search.filter.SearchFilter;
+
+
+@Service
+public class EWSIntegration {
+
+    private final EWSProperties properties;
+
+    private final EWSMapper mapper;
+
+    private final FolderView folderView = new FolderView(10);
+
+    private final ExchangeService service = new ExchangeService(ExchangeVersion.Exchange2010_SP2);
+
+    private final Logger log = LoggerFactory.getLogger(EWSIntegration.class);
+
+    public EWSIntegration(final EWSProperties properties, final EWSMapper mapper) {
+        this.properties = properties;
+        this.mapper = mapper;
+    }
+
+    public List<Email> pageThroughEntireInbox(final String requestedDestinationFolder) {
```

Comment:
Question: 
How is the value of `requestedDestinationFolder` decided? 
Should it be a part of the EWSProperties (i.e. do we want the value configurable per mailbox)?